spikeinterface/extractors/neoextractors/biocam.py

```diff
@@ -0,0 +1,41 @@
+from .neobaseextractor import NeoBaseRecordingExtractor, NeoBaseSortingExtractor
+
+
+class BiocamRecordingExtractor(NeoBaseRecordingExtractor):
+    """
+    Class for reading data from a Biocam file from 3Brain.
+
+    Based on neo.rawio.BiocamRawIO
+
+    Parameters
+    ----------
+    file_path: str
+
+    stream_id: str or None
+
+    mea_pitch: float or None
+
+    """
+    mode = 'file'
+    NeoRawIOClass = 'BiocamRawIO'
+
+    def __init__(self, file_path, stream_id=None, mea_pitch=None):
+        neo_kwargs = {'filename': str(file_path)}
+        if mea_pitch is not None:
+            neo_kwargs.update({'mea_pitch': mea_pitch})
+        NeoBaseRecordingExtractor.__init__(self, stream_id=stream_id, **neo_kwargs)
+
+        # load locations from neo array annotations
```

Comment:
You should make a location reader in probe interface alessio with contact size, shape, ...
No ?